packages/toolbox/src/commands/chainlink/listAggregators.ts

```diff
@@ -10,5 +10,16 @@ export const builder = (): yargs.Argv<IOptions> =>
       type: 'string',
       desc: 'Network for which to list aggregators (mainnet, rinkeby, private)',
     },
+    web3Url: {
+      demand: false,
+      type: 'string',
+      desc: 'A Web3 RPC url. If not provided will use the default',
+    },
+    lastBlock: {
```

Comment:
NB: this trick is not enough for Matic, there are already >100'000 blocks since the contract got released. It means we cannot compute paths automatically, unless we only update inputs / outputs / aggregators by full batch. Acceptable for now, considering the low costs, but might lead to errors.